buildenv/jenkins/openjdk10_ppc64le_linux

```diff
@@ -3,7 +3,7 @@
 	Set parameter TARGET(openjdk, system, external, perf, jck etc.).
 	Set UPSTREAM_JOB_NAME(version_build_arch_os, for example: openjdk8_build_x86-64_linux)
 	Set JVM_VERSION(openjdk8, openjdk8-openj9, openjdk9, openjdk8-openj9, etc.*/
-LABEL='linux&&ppc64le&&test'
+LABEL='linux&&ppc64le&&test||ci.project.openj9&&sw.os.linux&&hw.arch.ppc'
 
```

Comment:
this could become simply: sw.os.linux&&hw.arch.ppc

Reply:
The label should be `hw.arch.ppcle` not `hw.arch.ppc`. Please also check for jdk8, 9 and 10 ppcle file.

Reply:
yes, we should decide sw.os.linux&&hw.arch.ppc&&hw.endian.le or sw.os.linux&&hw.arch.ppcle  and then ensure correct labels used.  I was going based on the proposed schema here: https://github.com/smlambert/openjdk-infrastructure/blob/labels/docs/jenkinslabels.md  - but the proposal can be updated... note if we follow the proposal, then also implies we go with sw.os.aix.7_1  vs. sw.os.aix7.1